src/meta/proto-conv/tests/it/main.rs

```diff
@@ -34,3 +34,5 @@ mod v024_table_meta;
 mod v025_user_stage;
 mod v026_schema;
 mod v027_schema;
+mod v028_schema;
+mod v029_schema;
```

Comment:
Is there a `v028_schema` file added?

Reply:
yes, 0028 is `2023-02-13: Add: user.proto/UserDefinedFileFormat`